docs/getting-started/github-apps/note--registration.rst

```diff
@@ -1 +1 @@
-.. note:: You do not need an account to search for workflows on Dockstore or to launch them with our compute partners. However, to register content on Dockstore, you must :doc:`have an account on Dockstore and link the necessary third-party accounts </getting-started/register-on-dockstore>`. Once this is done you can register workflows from the My Workflows page, tools from the My Tools page, or services from the My Services page.
\ No newline at end of file
+.. note:: You do not need an account to search for workflows on Dockstore or to launch them with our compute partners. However, to register content on Dockstore, you must :doc:`have an account on Dockstore and link the necessary third-party accounts </getting-started/register-on-dockstore>`. Once this is done you can register workflows from the My Workflows page, tools from the My Tools page, services from the My Services page, or notebooks from the My Workflows page (currently in beta mode).
```

Comment:
We call it preview, not beta. I don't think we're allowed to have beta for compliance reasons. Just flagging it here, but it's in other locations.